src/main/java/it/pagopa/afm/marketplacebe/controller/TouchpointController.java

```diff
@@ -0,0 +1,105 @@
+
+@RestController
+public class TouchpointController {
+
+    @Autowired
+    private TouchpointService touchpointService;
+
+    @Operation(summary = "Get touchpoints", security = {@SecurityRequirement(name = "ApiKey")}, tags = {"Touchpoint",})
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "OK", content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE, schema = @Schema(implementation = Touchpoints.class))),
+            @ApiResponse(responseCode = "400", description = "Bad Request", content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE, schema = @Schema(implementation = ProblemJson.class))),
+            @ApiResponse(responseCode = "401", description = "Unauthorized", content = @Content(schema = @Schema())),
+            @ApiResponse(responseCode = "404", description = "Not Found", content = @Content(schema = @Schema())),
+            @ApiResponse(responseCode = "429", description = "Too many requests", content = @Content(schema = @Schema())),
+            @ApiResponse(responseCode = "500", description = "Service unavailable", content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE, schema = @Schema(implementation = ProblemJson.class)))})
+    @GetMapping(
+            value = "/touchpoints",
+            produces = {MediaType.APPLICATION_JSON_VALUE}
+    )
+    public Touchpoints getTouchpoints(
+            @Positive @Parameter(description = "Number of items for page. Default = 50") @RequestParam(required = false, defaultValue = "50") Integer limit,
+            @PositiveOrZero @Parameter(description = "Page number. Page number value starts from 0. Default = 1") @RequestParam(required = false, defaultValue = "1") Integer page){
+        return touchpointService.getTouchpoints();
+    }
+
+    @Operation(summary = "Get touchpoint", security = {@SecurityRequirement(name = "ApiKey")}, tags = {"Touchpoint",})
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "OK", content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE, schema = @Schema(implementation = Touchpoint.class))),
+            @ApiResponse(responseCode = "400", description = "Bad Request", content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE, schema = @Schema(implementation = ProblemJson.class))),
+            @ApiResponse(responseCode = "401", description = "Unauthorized", content = @Content(schema = @Schema())),
+            @ApiResponse(responseCode = "404", description = "Not Found", content = @Content(schema = @Schema())),
+            @ApiResponse(responseCode = "429", description = "Too many requests", content = @Content(schema = @Schema())),
+            @ApiResponse(responseCode = "500", description = "Service unavailable", content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE, schema = @Schema(implementation = ProblemJson.class)))})
+    @GetMapping(
+            value = "/touchpoint/{id}",
```

Comment:
I think it is better to adopt the resource name at plural according to [richardson maturity model](https://martinfowler.com/articles/richardsonMaturityModel.html#level1)

Reply:
Nice point, updated with a0e9766 👍